monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/home/GroupSelector.js

```diff
@@ -31,30 +33,24 @@ export default function GroupSelector() {
     },[])
 
     return (
-        <Box component="span" m={1}>
-            <Box style={{padding: '13px'}}>Group ID  <SelectComponent groupList={groupList}/></Box>
-        </Box>
+        <SelectComponent groupList={groupList}/>
     );
 }
 
 function SelectComponent(props) {
-
     var options = props.groupList;
     const dispatch = useDispatch();
-    let select;
-
-    if(options.length > 0) {
-        select = <Select native
-                    label="Group Id"
-                    onChange={(e) => dispatch(changeGroup(e.target.value))}
-                    style={{ textDecoration: 'none' , color: '#fff'}} >
-                    {options.map((option) => (
-                        <option value={option.value}>{option.label}</option>
-                    ))}
-                </Select>
-    } else {
-        select = <Select native
-                    style={{ textDecoration: 'none' , color: '#fff'}} />
-    }
-    return select;
+    return <FormControl style={{width: 150}}>
+    <InputLabel id="group-id-select-label">Group Id</InputLabel>
+    <Select
```

Comment:
<img width="321" alt="Screenshot 2021-02-16 at 10 07 39" src="https://user-images.githubusercontent.com/38608578/108018826-cf1d0180-703e-11eb-94a8-89df44ac9d68.png">

Select fields are not aligned properly. Shall we fix this?